src/languages/en-US.ts

```diff
@@ -3735,10 +3735,11 @@ export default class extends Language {
 		COMMAND_HOROSCOPE_TITLES: {
 			DAILY_HOROSCOPE: sign => `Daily horoscope for ${sign}`,
 			METADATA_TITLE: 'Metadata',
-			METADATA: (intensity, keywords, mood) => ([
+			METADATA: (intensity, keywords, mood, rating) => ([
 				`**Intensity:** ${intensity}`,
-				`**Keywords:** ${keywords}`,
-				`**Mood:** ${mood}`
+				`**Keywords:** ${this.list(keywords, 'and')}`,
+				`**Mood:** ${mood}`,
+				`**Rating:** ${Emojis.FilledStar.repeat(rating)}${Emojis.EmptyStar.repeat(5 - rating)}`
```

Comment:
The stars should be made from the code, not from the language files. Doing this will make it harder to switch to i18next.